privaterelay/templates/newlanding/a/plans.html

```diff
@@ -32,15 +33,15 @@
                             <li>{% ftlmsg 'landing-pricing-premium-feature-4' %}</li>
                         </ul>
                     </div>
-                    <a class="mzp-c-button mzp-t-product mzp-t-secondary mzp-t-lg">
+                    <a class="mzp-c-button mzp-t-product mzp-t-secondary mzp-t-lg" href="{% premium_subscribe_url accept_language %}" target="_blank" rel="noopener noreferrer">
                         {% ftlmsg 'nav-profile-sign-up' %}
                     </a>
                 </div>
             </div>
     
             <div class="c-pricing-text">
                 <div class="c-text">
-                    <h2 class="c-main-header">{% ftlmsg 'landing-pricing-headline' monthly_price='$0.99 USD' %}</h2>
```

Comment:
Same comment as above. the `USD` is part of the design. I'm not sure if that's a legal requirement or not?